llvm/lib/SandboxIR/Tracker.cpp

```diff
@@ -160,6 +160,34 @@ void CallBrInstSetIndirectDest::dump() const {
 }
 #endif
 
+SetVolatile::SetVolatile(Instruction *I, bool WasBool, Tracker &Tracker)
+    : IRChangeBase(Tracker), Inst(I) {
```

Comment:
drop `, Inst(I)` we can use `I` directly from the constructor's arguments.

Reply:
Fixed it.